tools/docker-compose-minikube/minikube/templates/bootstrap_minikube.py.j2

```diff
@@ -0,0 +1,42 @@
+# Create Openshift/Kubernetes credential for Minikube
+# This script gets called by the bootstrap_development process
+# awx-manage shell_plus --quiet < bootstrap_minikube.py
+
+from awx.main.utils.encryption import encrypt_field
+
+NAME = 'Minikube'
+
+POD_SPEC = """apiVersion: v1
+kind: Pod
+metadata:
+  namespace: default
```

Comment:
If you want to be very consistent, you could figure out how to pass in what the `namespace` should be here based on `minikube_service_account_namespace` in the playbook, because if it is something other than default the user will get permission denied because here is where we specify what namespace to deploy the pod in

Reply:
Yes, that is a good point. :)